doc/7-systems/snes.md

```diff
@@ -25,55 +25,153 @@ Furnace also allows the SNES to use wavetables (and the wavetable synthesizer) i
+|   `06` | 260 msec |  `06` |  74 msec |    `06` |  7/8  |    `06` |   12 sec
+|   `07` | 160 msec |  `07` |  37 msec |    `07` |   1   |    `07` |  9.4 sec
+|   `08` |  96 msec |       |          |         |       |    `08` |  7.1 sec
+|   `09` |  64 msec |       |          |         |       |    `09` |  5.9 sec
+|   `0A` |  40 msec |       |          |         |       |    `0A` |  4.7 sec
+|   `0B` |  24 msec |       |          |         |       |    `0B` |  3.5 sec
+|   `0C` |  16 msec |       |          |         |       |    `0C` |  2.9 sec
+|   `0D` |  10 msec |       |          |         |       |    `0D` |  2.4 sec
+|   `0E` |   6 msec |       |          |         |       |    `0E` |  1.8 sec
+|   `0F` |   0 msec |       |          |         |       |    `0F` |  1.5 sec
+|        |          |       |          |         |       |    `10` |  1.2 sec
+|        |          |       |          |         |       |    `11` | 880 msec
+|        |          |       |          |         |       |    `12` | 740 msec
+|        |          |       |          |         |       |    `13` | 590 msec
+|        |          |       |          |         |       |    `14` | 440 msec
+|        |          |       |          |         |       |    `15` | 370 msec
+|        |          |       |          |         |       |    `16` | 290 msec
+|        |          |       |          |         |       |    `17` | 220 msec
+|        |          |       |          |         |       |    `18` | 180 msec
+|        |          |       |          |         |       |    `19` | 150 msec
+|        |          |       |          |         |       |    `1A` | 110 msec
+|        |          |       |          |         |       |    `1B` |  92 msec
+|        |          |       |          |         |       |    `1C` |  74 msec
+|        |          |       |          |         |       |    `1D` |  55 msec
+|        |          |       |          |         |       |    `1E` |  37 msec
+|        |          |       |          |         |       |    `1F` |  18 msec
+
+reference: [Super Famicom Development Wiki](https://wiki.superfamicom.org/spc700-reference#dsp-voice-register:-adsr-1097)
+
+## gain
+
+value | linear inc. | bent line inc. | linear dec. | exponent dec.
+----: | ----------: | -------------: | ----------: | ------------:
+ `00` |           ∞ |              ∞ |           ∞ |             ∞
+ `01` |     4.1 sec |        7.2 sec |     4.1 sec |        38 sec
+ `02` |     3.1 sec |        5.4 sec |     3.1 sec |        28 sec
+ `03` |     2.6 sec |        4.6 sec |     2.6 sec |        24 sec
+ `04` |     2.0 sec |        3.5 sec |     2.0 sec |        19 sec
+ `05` |     1.5 sec |        2.6 sec |     1.5 sec |        14 sec
+ `06` |     1.3 sec |        2.3 sec |     1.3 sec |        12 sec
+ `07` |     1.0 sec |        1.8 sec |     1.0 sec |       9.4 sec
+ `08` |    770 msec |        1.3 sec |    770 msec |       7.1 sec
```

Comment:
Replace "msec" with "ms", and "sec" with "s". No space.

Reply:
Done!